1주차/4796/4796_python_후디.py

```diff
@@ -0,0 +1,15 @@
+case_list = []
+
+index = 0
+while True:
+    case = list(map(int, input().split(" ")))
+    if case[0] == case[1] == case[2] == 0:
+        break
+
+    l, p, v = case
+
+    day = v // p * l + min(l, v - v // p * p)
```

Comment:
```suggestion
    day = v // p * l + min(l, v - (v // p) * p)
```
v - v를 보고 멈칫했네요ㅎㅎ.  연산자 우선 순위가 있지만 이런 경우에 괄호로 구분해주면 가독성이 나아질 수 있겠죠?

Reply:
실제 코딩테스트를 생각했을때 당황하지 않도록 이런 가독성 챙겨주는 것 꼭 필요하겠네요 👍